libsplinter/src/admin/service/shared.rs

```diff
@@ -765,53 +861,49 @@ impl AdminServiceShared {
         payload: CircuitManagementPayload,
         message_sender: String,
     ) -> Result<(), ServiceError> {
-        let mut unauthorized_peers = vec![];
         let mut missing_protocol_ids = vec![];
+        let mut pending_peers = vec![];
+        let mut added_peers: Vec<String> = vec![];
+        let mut pending_members = vec![];
         for node in payload
             .get_circuit_create_request()
             .get_circuit()
             .get_members()
         {
             if self.node_id() != node.get_node_id() {
-                if self.auth_inquisitor().is_authorized(node.get_node_id()) {
-                    if self
-                        .service_protocols
-                        .get(&admin_service_id(node.get_node_id()))
-                        .is_none()
-                    {
-                        // we will always have the network sender at this point
-                        if let Some(ref network_sender) = self.network_sender {
-                            debug!("Sending service protocol request to {}", node.get_node_id());
-                            let mut request = ServiceProtocolVersionRequest::new();
-                            request.set_protocol_min(ADMIN_SERVICE_PROTOCOL_MIN);
-                            request.set_protocol_max(ADMIN_PROTOCOL_VERSION);
-                            let mut msg = AdminMessage::new();
-                            msg.set_message_type(
-                                AdminMessage_Type::SERVICE_PROTOCOL_VERSION_REQUEST,
-                            );
-                            msg.set_protocol_request(request);
-
-                            let envelope_bytes = msg.write_to_bytes()?;
-                            network_sender
-                                .send(&admin_service_id(node.get_node_id()), &envelope_bytes)?;
+                debug!("Connecting to node {:?}", node);
```

Comment:
Should we use the phrase "Connecting to" here?  Perhaps "Referencing", so that the actual connection state isn't confusing in the logs.

Reply:
updated